github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml

```diff
@@ -66,7 +66,7 @@ periodics:
                 latest_version=$(curl --fail -s https://api.github.com/repos/kubevirt/common-instancetypes/releases | jq -r '.[] | select(.target_commitish == '\""${COMMON_INSTANCETYPES_BRANCH}"\"') | .tag_name' | head -n1)
                 description="Automated update of common-instancetypes bundles to ${latest_version}\n\n\\\`\\\`\\\`release-note\nUpdated common-instancetypes bundles to ${latest_version}\n\\\`\\\`\\\`"
 
-                git-pr.sh -c "cd ../kubevirt && hack/common-instancetypes/bump.sh ${COMMON_INSTANCETYPES_BRANCH}" -d "echo -e \"${description}\"" -r kubevirt -b update-common-instancetypes-main -T ${KUBEVIRT_BRANCH}
+                git-pr.sh -c "cd ../kubevirt && hack/bump-common-instancetypes.sh ${COMMON_INSTANCETYPES_BRANCH}" -d "echo -e \"[${KUBEVIRT_BRANCH}] ${description}\"" -r kubevirt -b update-common-instancetypes-${KUBEVIRT_BRANCH} -T ${KUBEVIRT_BRANCH}
```

Comment:
`hack/bump-common-instancetypes.sh` doesn't exist on `release-1.5` thanks to [6c26bf70e6fd40e8b622570a32d6f8cb1c4a3538](https://github.com/kubevirt/kubevirt/commit/6c26bf70e6fd40e8b622570a32d6f8cb1c4a3538) 

```
$ git branch --contains 6c26bf70e6fd40e8b622570a32d6f8cb1c4a3538 | grep release
  release-1.5
```